docs/source/usage/tool_calling.md

```diff
@@ -170,6 +170,10 @@ Recommended flags: `--tool-call-parser granite --chat-template examples/tool_cha
 
 `examples/tool_chat_template_granite.jinja`: this is a modified chat template from the original on Huggingface. Parallel function calls are supported.
 
+* `ibm-granite/granite-3.1-8b-instruct`
+
+Recommended flags: `--tool-call-parser granite`
+
```

Comment:
It would be nice to leave a note that the chat template from the model should be used, no need to specify otherwise